app/javascript/components/list-icon/component.spec.js

```diff
@@ -0,0 +1,14 @@
+import { mountedComponent, screen } from "../../test-utils";
+
+import ListIcon from "./component";
+
+describe("<ListIcon />", () => {
+  it("renders correct icon", () => {
+    mountedComponent(<ListIcon icon="cases" />);
+    expect(screen.getByText((content, element) => element.tagName.toLowerCase() === "svg")).toBeInTheDocument();
```

Comment:
For these, is it possible to add a test-id with the value of the icon prop in the ListIcon component and check the value of the test-id instead. Checking if a svg returns does not tell us which icons was returned.

Reply:
Done